homeworks/Yaroslav.Melnyk_Maritodd/homework_3/task2.js

```diff
@@ -0,0 +1,25 @@
+const readline = require('readline');
+
+const rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout,
+});
+
+function func(i) {
+    if (i === 1) {
+        return true;
+    }
+    if (i === 0) {
+        return false;
+    }
+    return func(i / 2);
+}
```

Comment:
what if i will never be equal to 0 or 1?

Reply:
> what if i will never be equal to 0 or 1?
i fixed